workflows/wf2/4-get-options-map.js

```diff
@@ -1,153 +1,40 @@
-  OZViJk8FPVd: 'c2664992-8a5a-4a6d-9238-5df591307d55', //Has the patient had thoughts of death or suicide?
-};
-
-const mhgapFollowup = {
-  //mhgap followup - see below formMap for programStage and form id
-  //yUT7HyjWurN: 'encounter-date', // encounterDate
-  CcA8pc2YqWz: 'd9454e9c-6e3c-45ab-8a9a-834a9353ae11', //Session number
-  G0hLyxqgcO7: '54e8c1b6-6397-4822-89a4-cf81fbc68ce9', //The patient did not come
-  tloy0Bd9qDf: 'd7410cd3-29be-4f8b-93d6-eb4de005db29', //Number of appointments missed
-  AZUFlZzPN6V: '278d3d1e-c02a-4db1-8ab3-8db3b82eb9b5', //Patient rescheduled
-  HNZHetAtowR: 'd34d5e93-03d6-494e-8f4c-2d7221227162', //Reason for missed appointment
-  t4HoHWCYdvm: '790b41ce-e1e7-11e8-b02f-0242ac130002', //Reason for missed appointment - If other, specify
-  MDbDqs5Mwow: '22809b19-54ca-4d88-8d26-9577637c184e', //Clinical diagnosis
-  EYIPzqO7mpZ: '819f79e7-b9af-4afd-85d4-2ab677223113', //Clinical diagnosis - If other, specify
-  faQI7NN3hyp: '5f3d618e-5c89-43bd-8c79-07e4e98c2f23', //PHQ-9 score
-  yTFUtaFJ1QU: 'a1a75011-0fef-460a-b666-dda2d171f39b', //CGI-S score
-  C5XtlggtVmd: 'f94de17e-9771-4711-aabb-c5bb0c022be2', //CGI-I score
-  AiUr0KZgS9e: 'dbf35880-d18e-4f15-ad9c-97b5b053e14a', //Adherence to treatment - Is the patient taking the medication as prescribed?
-  OBAHc3Rljln: 'dc072a1a-20d8-4512-9e57-3585a07cb074', //Adherence to treatement - Details if needed
-  WKDFgUTUHWl: '5f6e245c-83fc-421b-8d46-061ac773ae71', //Follow-up required
-};
-
-const mhpssClosure = {
-  //mhpss closure - see below formMap for programStage and form id
-  //yUT7HyjWurN: 'encounter-date', // encounterDate
-  RoBHeg55Mle: '8a9d2ace-372f-4413-a1c4-8999f55e2de2', //Closure date
-  TqjvJA0GUOA: '401b2df0-e26f-4f49-9165-6e20a1b8865a', //Main type of consultation
-  Lht8GKTAHu8: '9e861ef1-e07c-4955-9650-2ebac3138fc3', //Type of closure
-  tvQUeCICtFB: '93eb9716-6866-4d13-9b8f-59c0a7605a11', //Patient referred to
-  pQG72ee2lkl: '90b3d09c-d296-44d2-8292-8e04377fe027', //MHOS
-  tsFOVnlc6lz: '6545b874-f44d-4d18-9ab1-7a8bb21c0a15', //PHQ-9
-  iEdamWl0wzQ: 'c64dfacd-2fb5-464d-83a9-ebb9eb8d3ab3', //CGI-I
-  c4TiPRUQ56A: '77f1a782-24af-40ba-bac1-b97d9a080f40', //CGI-S
-};
-
 // state.mhgapClosure = { //Waiting on form
 //   yUT7HyjWurN: 'encounter-date', // encounterDate
 // };
-
 get(
-  'https://gist.githubusercontent.com/aleksa-krolls/2f309595110239cfcc8251017cf11592/raw/43e11ddf77a17e393b4ce756fcec90e2c4339ca9/msf_wf2_optionsMap.json'
-); //ak's gist
+  'https://raw.githubusercontent.com/OpenFn/openfn-lime-pilot/refs/heads/main/metadata/metadata_mapping.json'
+);
 
 fn(state => {
-  const { data, response, references, ...next } = state;
-  next.optsMap = JSON.parse(data);
-  // console.log(JSON.stringify(next.optsMap, null, 2), 'Options Map');
-  return next;
+  const {
+    optionSets,
+    f01MhpssBaseline,
+    f02MhpssFollowUp,
+    f03MhgapBaseline,
+    f04MhgapFollowUp,
+    f05MhpssClosure,
+  } = JSON.parse(state.data);
```

Comment:
Why do you have to JSON parse this?

If the HTTP adaptor isn't auto-parsing JSON, we need to raise an issue.

Reply:
I need to pass `parseAs: 'json'`